frontend/src/components/BrowserSession.tsx

```diff
@@ -90,7 +96,24 @@ const BrowserSession: React.FC<Props> = ({ session, isCurrent }) => {
     });
   };
 
-  const sessionName = isCurrent ? "Current browser session" : "Browser session";
+  const deviceInformation = parseUserAgent(data.userAgent || undefined);
+  const name =
+    sessionNameFromDeviceInformation(deviceInformation) || "Browser session";
+  const sessionName = isCurrent ? (
```

Comment:
Can you instead make a `current` prop on the `Session` component, so that it can be displayed on top of the session name and not inline?

Reply:
<img width="405" alt="Screenshot 2023-09-05 at 12 33 20" src="https://github.com/matrix-org/matrix-authentication-service/assets/3055605/15176b8c-6bee-4921-9b14-89acc52ed4b7">